src/webui/www/private/shareratio.html

```diff
@@ -92,6 +96,7 @@
                     ratioLimitValue = document.getElementById("setRatio").checked ? document.getElementById("ratio").value : -1;
                     seedingTimeLimitValue = document.getElementById("setTotalMinutes").checked ? document.getElementById("totalMinutes").value : -1;
                     inactiveSeedingTimeLimitValue = document.getElementById("setInactiveMinutes").checked ? document.getElementById("inactiveMinutes").value : -1;
+                    shareLimitActionValue = document.getElementById("action").value;
```

Comment:
It would be nice to have a variable for `document.getElementById("action")` so you don't need to call it more than once.

Reply:
I mentioned it because overall we have too many `document.getElementById` calls in the code base. And most of them are just due to bad coding style from the past. Recently we start to cut down redundant `getElementById` calls in reviews and try to not introduce more of them from now on.